### DISCUSSION TITLE
https://vuepress-theme-hope.github.io/v2/zh/guide/markdown/external.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#594] [Announcements]
# https://vuepress-theme-hope.github.io/v2/zh/guide/markdown/external.html

除了在 Markdown 内部使用新语法来增强内容外，你也可以使用 &lt;iframe&gt; 或 &lt;img&gt; 从外部引入链接来增强你的 Markdown 内容。

https://vuepress-theme-hope.github.io/v2/zh/guide/markdown/external.html

**Comment by leoleor** [comment]
[文章](https://www.zhihu.com/question/19554716​)

点开链接后变成https://www.zhihu.com/question/19554716%E2%80%8B
